samples/crypto/psa_tls/sample.yaml

```diff
@@ -35,36 +38,43 @@ tests:
       - nrf9160dk/nrf9160/ns
       - nrf9161dk/nrf9161
       - nrf9161dk/nrf9161/ns
+      - nrf9151dk/nrf9151
+      - nrf9151dk/nrf9151/ns
     tags: ci_build cc3xx_oberon dtls
   sample.psa_tls.tls_client.ecdsa.cc3xx_oberon:
     build_only: true
     extra_args: >
       OVERLAY_CONFIG="overlays/client.conf;overlays/ecdsa.conf;overlays/cc3xx-oberon-psa.conf"
     platform_allow: >
       nrf5340dk/nrf5340/cpuapp nrf9160dk/nrf9160 nrf5340dk/nrf5340/cpuapp/ns nrf9160dk/nrf9160/ns
-      nrf9161dk/nrf9161 nrf9161dk/nrf9161/ns
+      nrf9161dk/nrf9161 nrf9161dk/nrf9161/ns nrf9151dk/nrf9151 nrf9151dk/nrf9151/ns
     integration_platforms:
       - nrf5340dk/nrf5340/cpuapp
       - nrf5340dk/nrf5340/cpuapp/ns
       - nrf9160dk/nrf9160
       - nrf9160dk/nrf9160/ns
       - nrf9161dk/nrf9161
       - nrf9161dk/nrf9161/ns
+      - nrf9151dk/nrf9151
+      - nrf9151dk/nrf9151/ns
     tags: ci_build cc3xx_oberon
   sample.psa_tls.dtls_client.ecdsa.cc3xx_oberon:
     build_only: true
     extra_args: >
       OVERLAY_CONFIG="overlays/client.conf;overlays/ecdsa.conf;overlays/cc3xx-oberon-psa.conf;overlays/dtls.conf"
     platform_allow: >
       nrf5340dk/nrf5340/cpuapp nrf9160dk/nrf9160 nrf5340dk/nrf5340/cpuapp/ns
-      nrf9160dk/nrf9160/ns nrf9161dk/nrf9161 nrf9161dk/nrf9161/ns
+      nrf9160dk/nrf9160/ns nrf9161dk/nrf9161 nrf9161dk/nrf9161/ns nr9151dk/nrf9151
```

Comment:
```suggestion
      nrf9160dk/nrf9160/ns nrf9161dk/nrf9161 nrf9161dk/nrf9161/ns nrf9151dk/nrf9151
```

Reply:
Good catch! I wonder if this was caused by copilot autocomplete.